### ISSUE TITLE
Column sort icons don't show up on Solarized theme

### ISSUE THREAD

**Issue by mhkeller** [finos/perspective#2359]
### Steps to Reproduce:
 1. Import the solarized.css theme
 2. Set that theme via `.restore`
 3. Click on any of the column headers

### Expected Result:
I'd expect to see the up and down arrows

### Actual Result:
I see a grey rectangle

### Environment:
Perspective: 2.5.1
Node: v18.12.1
MacOS 12.6.8

### Additional Context:
Here's a reproduction based off the esbuild-example in the main perspective repo: https://github.com/mhkeller/perspective-solarized-repro
<img width="809" alt="Screen Shot 2023-09-05 at 11 02 21 PM" src="https://github.com/finos/perspective/assets/498744/0abe8327-a209-480c-bc36-9c35fe0ff1b5">

This issue doesn't appear if I import the theme.css file via a normal `<link>` tag.

**Comment by texodus** [comment]
There is a note on this in [the Perspective docs sections on Themes](https://perspective.finos.org/docs/js/#theming):

```javascript
viewer.resetThemes(["Solarized"]);
```

**Comment by texodus** [comment]
Thanks @mhkeller this is a bug. You can work around it by importing the regular `themes.css` and just setting the theme through the restore API:

```javascript
viewer.restore({theme: "Solarized"})
```

**Comment by Vedashree11** [comment]
viewer.restore({theme: "Solarized"})

**Comment by texodus** [comment]
This is fixed and will be in `v2.7.1`

**Comment by mhkeller** [comment]
Thanks!
